delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/plugin/PluginsInitializer.scala

```diff
@@ -15,9 +19,28 @@ object PluginsInitializer {
   def apply(
       serviceModule: ModuleDef,
       pluginsDef: List[PluginDef]
-  ): IO[PluginInitializationError, (List[Plugin], Locator)] = {
+  )(implicit classLoader: ClassLoader): IO[PluginInitializationError, (List[Plugin], Locator)] = {
     val pluginsInfoModule = new ModuleDef {
-      make[List[PluginInfo]].from(pluginsDef.map(_.info))
+      make[List[PluginDescription]].from(pluginsDef.map(_.info))
+      make[RemoteContextResolution].from(
```

Comment:
It feels a little weird that something that is used also in the core to be initialized here